### ISSUE TITLE
Push skip-links to npm? 

### ISSUE THREAD

**Issue by elisechant** [govau/uikit#141]
Trying to install Skip links `yarn add @gov.au/skip-link`. Realise that it's not in npm?

**Comment by dominikwilkowski** [comment]
We are awaiting the a11y review. As soon as that is done and implemented we will go live with the uikit :)
_(I assume this is likely this Thursday.)_

**Comment by elisechant** [comment]
can't you push without a11y review!?

**Comment by dominikwilkowski** [comment]
No what we push needs to be a11y
That was my call and I stand to it. It doesn't have to be perfect but at very least it has to be a11y

**Comment by elisechant** [comment]
okie dokie
